### PR TITLE
Clarify empty folder field behaviour for S3 backups

### PR DESCRIPTION
**Pull request by suse-gabrielblais**
Added details to explicitly mention how to use the root directory of an S3 bucket for storing backups generated by the Rancher Backups operator.

### CODE REVIEW

File: docs/reference-guides/backup-restore-configuration/backup-configuration.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ The S3 storage location contains the following configuration fields:
...
 1. **Bucket Name**: The name of the S3 bucket where backup files will be stored.
 1. **Region** (optional): The AWS [region](https://aws.amazon.com/about-aws/global-infrastructure/regions_az/) where the S3 bucket is located. This field isn't needed for configuring MinIO.
-1. **Folder** (optional): The name of the folder in the S3 bucket where backup files will be stored. Nested folders (e.g., `rancher/cluster1`) are not supported.
+1. **Folder** (optional): The name of the folder in the S3 bucket where backup files will be stored. Nested folders (e.g., `rancher/cluster1`) are not supported. An empty value will store the backup files in the root folder of the S3 bucket.
```
Comment (martyav) [inline docs/reference-guides/backup-restore-configuration/backup-configuration.md:76]:
```suggestion
1. **Folder** (optional): The name of the folder in the S3 bucket where backup files will be stored. Nested folders (e.g., `rancher/cluster1`) are not supported. If this value is left empty, the default behavior is to store the backup files in the root folder of the S3 bucket.
```

File: docs/reference-guides/backup-restore-configuration/backup-configuration.md [hunk 2]
```diff
@@ -84,7 +84,7 @@ The S3 storage location contains the following configuration fields:
...
 | `credentialSecretNamespace` | The namespace of the secret containing the credentials to access S3. This directive is unnecessary if the nodes running your operator are in EC2 and set up with IAM permissions that allow them to access S3, as described in [this section.](#iam-permissions-for-ec2-nodes-to-access-s3) |  |
 | `bucketName` | The name of the S3 bucket where backup files will be stored. | ✓ |
-| `folder` | The name of the folder in the S3 bucket where backup files will be stored. Nested folders (e.g., `rancher/cluster1`) are not supported. | |
+| `folder` | The name of the folder in the S3 bucket where backup files will be stored. Nested folders (e.g., `rancher/cluster1`) are not supported. An empty value will store the backup files in the root folder of the S3 bucket. | |
```
Comment (martyav) [inline docs/reference-guides/backup-restore-configuration/backup-configuration.md:87]:
```suggestion
| `folder` | The name of the folder in the S3 bucket where backup files will be stored. Nested folders (e.g., `rancher/cluster1`) are not supported. If this field is left empty, the default behavior is to store the backup files in the root folder of the S3 bucket. | |
```

File: docs/reference-guides/backup-restore-configuration/storage-configuration.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ You can choose to not have any operator-level storage location configured. If yo
 | Credential Secret | Choose the credentials for S3 from your secrets in Rancher. [Example](examples.md#example-credential-secret-for-storing-backups-in-s3). |
 | Bucket Name | Enter the name of the [S3 bucket](https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingBucket.html) where the backups will be stored. Default: `rancherbackups`. |
 | Region | The [AWS region](https://aws.amazon.com/about-aws/global-infrastructure/regions_az/) where the S3 bucket is located. |
-| Folder | The [folder in the S3 bucket](https://docs.aws.amazon.com/AmazonS3/latest/user-guide/using-folders.html) where the backups will be stored. |
+| Folder | The [folder in the S3 bucket](https://docs.aws.amazon.com/AmazonS3/latest/user-guide/using-folders.html) where the backups will be stored. An empty value will store the backup files in the root folder of the S3 bucket. |
```
Comment (martyav) [inline docs/reference-guides/backup-restore-configuration/storage-configuration.md:23]:
```suggestion
| Folder | The [folder in the S3 bucket](https://docs.aws.amazon.com/AmazonS3/latest/user-guide/using-folders.html) where the backups will be stored. If this field is left empty, the default behavior is to store the backup files in the root folder of the S3 bucket. |
```

File: versioned_docs/version-2.5/reference-guides/backup-restore-configuration/backup-configuration.md [hunk 4]
```diff
@@ -74,7 +74,7 @@ The S3 storage location contains the following configuration fields:
...
 1. **Bucket Name**: The name of the S3 bucket where backup files will be stored.
 1. **Region** (optional): The AWS [region](https://aws.amazon.com/about-aws/global-infrastructure/regions_az/) where the S3 bucket is located. This field isn't needed for configuring MinIO.
-1. **Folder** (optional): The name of the folder in the S3 bucket where backup files will be stored. Nested folders (e.g., `rancher/cluster1`) are not supported.
+1. **Folder** (optional): The name of the folder in the S3 bucket where backup files will be stored. Nested folders (e.g., `rancher/cluster1`) are not supported. An empty value will store the backup files in the root folder of the S3 bucket.
```
Comment (martyav) [inline versioned_docs/version-2.5/reference-guides/backup-restore-configuration/backup-configuration.md:77]:
```suggestion
1. **Folder** (optional): The name of the folder in the S3 bucket where backup files will be stored. Nested folders (e.g., `rancher/cluster1`) are not supported. If this field is left empty, the default behavior is to store the backup files in the root folder of the S3 bucket.
```

File: versioned_docs/version-2.5/reference-guides/backup-restore-configuration/backup-configuration.md [hunk 5]
```diff
@@ -85,7 +85,7 @@ The S3 storage location contains the following configuration fields:
...
 | `credentialSecretNamespace` | The namespace of the secret containing the credentials to access S3. This directive is unnecessary if the nodes running your operator are in EC2 and set up with IAM permissions that allow them to access S3, as described in [this section.](#iam-permissions-for-ec2-nodes-to-access-s3) |  |
 | `bucketName` | The name of the S3 bucket where backup files will be stored. | ✓ |
-| `folder` | The name of the folder in the S3 bucket where backup files will be stored. Nested folders (e.g., `rancher/cluster1`) are not supported. | |
+| `folder` | The name of the folder in the S3 bucket where backup files will be stored. Nested folders (e.g., `rancher/cluster1`) are not supported. An empty value will store the backup files in the root folder of the S3 bucket. | |
```
Comment (martyav) [inline versioned_docs/version-2.5/reference-guides/backup-restore-configuration/backup-configuration.md:88]:
```suggestion
| `folder` | The name of the folder in the S3 bucket where backup files will be stored. Nested folders (e.g., `rancher/cluster1`) are not supported. If this field is left empty, the default behavior is to store the backup files in the root folder of the S3 bucket. | |
```

File: versioned_docs/version-2.5/reference-guides/backup-restore-configuration/storage-configuration.md [hunk 6]
```diff
@@ -20,7 +20,7 @@ You can choose to not have any operator-level storage location configured. If yo
 | Credential Secret | Choose the credentials for S3 from your secrets in Rancher. [Example](./examples.md#example-credential-secret-for-storing-backups-in-s3). |
 | Bucket Name | Enter the name of the [S3 bucket](https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingBucket.html) where the backups will be stored. Default: `rancherbackups`. |
 | Region | The [AWS region](https://aws.amazon.com/about-aws/global-infrastructure/regions_az/) where the S3 bucket is located. |
-| Folder | The [folder in the S3 bucket](https://docs.aws.amazon.com/AmazonS3/latest/user-guide/using-folders.html) where the backups will be stored. |
+| Folder | The [folder in the S3 bucket](https://docs.aws.amazon.com/AmazonS3/latest/user-guide/using-folders.html) where the backups will be stored. An empty value will store the backup files in the root folder of the S3 bucket. |
```
Comment (martyav) [inline versioned_docs/version-2.5/reference-guides/backup-restore-configuration/storage-configuration.md:23]:
```suggestion
| Folder | The [folder in the S3 bucket](https://docs.aws.amazon.com/AmazonS3/latest/user-guide/using-folders.html) where the backups will be stored. If this field is left empty, the default behavior is to store the backup files in the root folder of the S3 bucket. |
```

File: versioned_docs/version-2.6/reference-guides/backup-restore-configuration/backup-configuration.md [hunk 7]
```diff
@@ -73,7 +73,7 @@ The S3 storage location contains the following configuration fields:
...
 1. **Bucket Name**: The name of the S3 bucket where backup files will be stored.
 1. **Region** (optional): The AWS [region](https://aws.amazon.com/about-aws/global-infrastructure/regions_az/) where the S3 bucket is located. This field isn't needed for configuring MinIO.
-1. **Folder** (optional): The name of the folder in the S3 bucket where backup files will be stored. Nested folders (e.g., `rancher/cluster1`) are not supported.
+1. **Folder** (optional): The name of the folder in the S3 bucket where backup files will be stored. Nested folders (e.g., `rancher/cluster1`) are not supported. An empty value will store the backup files in the root folder of the S3 bucket.
```
Comment (martyav) [inline versioned_docs/version-2.6/reference-guides/backup-restore-configuration/backup-configuration.md:76]:
```suggestion
1. **Folder** (optional): The name of the folder in the S3 bucket where backup files will be stored. Nested folders (e.g., `rancher/cluster1`) are not supported. If this field is left empty, the default behavior is to store the backup files in the root folder of the S3 bucket.
```

File: versioned_docs/version-2.6/reference-guides/backup-restore-configuration/backup-configuration.md [hunk 8]
```diff
@@ -84,7 +84,7 @@ The S3 storage location contains the following configuration fields:
...
 | `credentialSecretNamespace` | The namespace of the secret containing the credentials to access S3. This directive is unnecessary if the nodes running your operator are in EC2 and set up with IAM permissions that allow them to access S3, as described in [this section.](#iam-permissions-for-ec2-nodes-to-access-s3) |  |
 | `bucketName` | The name of the S3 bucket where backup files will be stored. | ✓ |
-| `folder` | The name of the folder in the S3 bucket where backup files will be stored. Nested folders (e.g., `rancher/cluster1`) are not supported. | |
+| `folder` | The name of the folder in the S3 bucket where backup files will be stored. Nested folders (e.g., `rancher/cluster1`) are not supported. An empty value will store the backup files in the root folder of the S3 bucket. | |
```
Comment (martyav) [inline versioned_docs/version-2.6/reference-guides/backup-restore-configuration/backup-configuration.md:87]:
```suggestion
| `folder` | The name of the folder in the S3 bucket where backup files will be stored. Nested folders (e.g., `rancher/cluster1`) are not supported. If this field is left empty, the default behavior is to store the backup files in the root folder of the S3 bucket. | |
```

File: versioned_docs/version-2.6/reference-guides/backup-restore-configuration/storage-configuration.md [hunk 9]
```diff
@@ -20,7 +20,7 @@ You can choose to not have any operator-level storage location configured. If yo
 | Credential Secret | Choose the credentials for S3 from your secrets in Rancher. [Example](examples.md#example-credential-secret-for-storing-backups-in-s3). |
 | Bucket Name | Enter the name of the [S3 bucket](https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingBucket.html) where the backups will be stored. Default: `rancherbackups`. |
 | Region | The [AWS region](https://aws.amazon.com/about-aws/global-infrastructure/regions_az/) where the S3 bucket is located. |
-| Folder | The [folder in the S3 bucket](https://docs.aws.amazon.com/AmazonS3/latest/user-guide/using-folders.html) where the backups will be stored. |
+| Folder | The [folder in the S3 bucket](https://docs.aws.amazon.com/AmazonS3/latest/user-guide/using-folders.html) where the backups will be stored. An empty value will store the backup files in the root folder of the S3 bucket. |
```
Comment (martyav) [inline versioned_docs/version-2.6/reference-guides/backup-restore-configuration/storage-configuration.md:23]:
```suggestion
| Folder | The [folder in the S3 bucket](https://docs.aws.amazon.com/AmazonS3/latest/user-guide/using-folders.html) where the backups will be stored. If this field is left empty, the default behavior is to store the backup files in the root folder of the S3 bucket. |
```

**Review by martyav — COMMENTED** [review]
Slight rephrase suggestion by docs team

**Comment by suse-gabrielblais** [comment]
> Slight rephrase suggestion by docs team

Thank you, applied the suggestions! Please let me know if there is anything else. =]

**Review by martyav — APPROVED** [review]

**Review by btat — APPROVED** [review]

**Pull request merged**